tests/test_server.py

```diff
@@ -0,0 +1,100 @@
+from unittest.mock import Mock, patch
+
+from flask import g
+import pytest
+
+from scripts import __version__, server
+
+# see the flask docs for an explanation of creating an example test client:
+# https://flask.palletsprojects.com/en/1.1.x/testing/#the-testing-skeleton
+
+# see also the docs on mocking the 'g' context object:
+# https://flask.palletsprojects.com/en/1.1.x/testing/#faking-resources
+
+
+@pytest.fixture
+def client():
+    # FIXME incomplete
+    server.app.config['TESTING'] = True
+
+    with server.app.test_client() as client, server.app.app_context():
```

Comment:
is this all we need to do to setup the client? I didn't totally understand what was happening in the flask docs (seemed related to db setup/teardown, which I don't think we need). if this is all we need we can remove the FIXME

Reply:
This worked (I think it was once I added the `yield`); I'll go ahead and remove the fixme